vms/avm/wallet_service.go

```diff
@@ -195,8 +163,7 @@ func (w *WalletService) SendMultiple(_ *http.Request, args *SendMultipleArgs, re
 		return err
 	}
 
-	utxos, err = w.update(utxos)
-	if err != nil {
+	if err = w.vm.walletService.update(utxos); err != nil {
```

Comment:
I think this call to `update` looks strange in the new code. I think we can remove this call by having `LoadUser` called in the wallet